src/leappto/cli.py

```diff
@@ -135,27 +142,64 @@ def _set_ssh_config(username, identity, use_sshpass=False):
         return use_sshpass, ssh_options
 
     class MigrationContext:
-        def __init__(self, target, ssh_settings, disk, forwarded_ports=None):
+
+        _SSH_CTL_PATH = '{}/.ssh/ctl'.format(os.environ['HOME'])
+        _SSH_CONTROL_PATH = '-o ControlPath="{}/%L-%r@%h:%p"'.format(_SSH_CTL_PATH)
+
+        def __init__(self, source, target, ssh_settings, disk, forwarded_ports=None, rsync_cp_backend=False):
+            self.source = source
             self.target = target
-            self.use_sshpass, self.target_cfg = ssh_settings
+            self.use_sshpass, self.ssh_cfg = ssh_settings
```

Comment:
done, I think we have to refactor (or split) that class, I didn't want to do this in that PR.